### PR TITLE
Add namespace for service-account.yaml

### PR DESCRIPTION
**Pull request by carlochessamp**
:gear: **Issue**

*Ticket/s*:   

:gear: **Change** 

I have added the namespace metadata to an object

Change Type: 

AC:

:white_check_mark: **Fix**

:question: **Tests**

- [ ] Tests for new feature and update for changes
- [ ] Linting passes and/or formatting matches the standard of the repo

🗒️ **Documentation**

- [ ] Updated related documentation (if applicable).
- [ ] Updated Change log (if exists)

### CODE REVIEW

**Comment by carlochessamp** [comment]
Hello @Stuart133 I have this little fix for the helm chart

**Comment by ryanwohara** [comment]
@carlochessamp Thank you for the contribution. Can you help us understand in which circumstance this is required? We'd like to fully understand it, if only for documentation for the next release.

**Comment by carlochessamp** [comment]
@ryanwohara recently I was deploying this helm chart in a k8s cluster where the Circleci Container runner wasn't deployed to the default namespace later I notice that the service account was missing because it was created on the default namespace that the reason I'm suggestion to specify the service account namespace in the template :D

**Review by atulsingh0 — APPROVED** [review]

**Comment by ryanwohara** [comment]
@carlochessamp Can you share the install command you ran? Perhaps there is a problem with our docs?

**Comment by carlochessamp** [comment]
> @carlochessamp Can you share the install command you ran? Perhaps there is a problem with our docs?

I can't remember. It was done using terraform-helm integration resource so I don't have the details of that execution

File: templates/service-account.yaml [hunk 1]
```diff
@@ -33,6 +33,7 @@ kind: Secret
 metadata:
   name: {{ .Values.logging.serviceAccount.secret.name }}
   annotations:
+    namespace: {{ $namespace }}
```
Comment (christian-stephen) [inline templates/service-account.yaml:36]:
```suggestion
metadata:
  name: {{ .Values.logging.serviceAccount.secret.name }}
  namespace: {{ $namespace }}
```

**Review by christian-stephen — CHANGES_REQUESTED** [review]

**Review by christian-stephen — APPROVED** [review]

**Comment by carlochessamp** [comment]
Hello guys! friendly ping about this PR :D @christian-stephen @atulsingh0

**Comment by jrahme-cci** [comment]
Sorry for the delay @carlochessamp , I just pushed a changelog entry to your branch and will get this merged now.

**Pull request merged**